soh/soh/Enhancements/randomizer/3drando/spoiler_log.cpp

```diff
@@ -739,12 +728,23 @@ const char* SpoilerLog_Write(int language) {
     }
 
     std::string jsonString = jsonData.dump(4);
+    std::ostringstream fileNameStream;
+    for (int i = 0; i < Settings::hashIconIndexes.size(); i ++) {
+        if (i) {
+            fileNameStream << '-';
+        }
+        if (Settings::hashIconIndexes[i] < 10) {
+            fileNameStream << '0';
+        }
+        fileNameStream << std::to_string(Settings::hashIconIndexes[i]);
+    }
+    std::string fileName = fileNameStream.str();
```

Comment:
i feel like it'd be nice if we didn't have to have this logic in here and in `menu.cpp`, but we can make an issue to figure out the best way to clean up that copypasta later